scripts/setup.cfg

```diff
@@ -48,4 +48,5 @@ console_scripts =
     check-user = owners.checkuser:main
     metrics = metrics.metrics:main
     get-verify-params = report.get_verify_params:main
+    pushowners=pushowners.owners:main
```

Comment:
I think this  new python scripts should be under the script/metrics directory. We have too many directories already and the names gets confusing.

Reply:
So as per the below comments we need to move the scripts to scripts/owners directory.

Reply:
Sure, makes sense